public/icomoon/style.css

```diff
@@ -1,18 +1,20 @@
 @font-face {
-  font-family: 'icomoon';
-  src:  url('fonts/icomoon.eot?nj6lan');
-  src:  url('fonts/icomoon.eot?nj6lan#iefix') format('embedded-opentype'),
-    url('fonts/icomoon.ttf?nj6lan') format('truetype'),
-    url('fonts/icomoon.woff?nj6lan') format('woff'),
-    url('fonts/icomoon.svg?nj6lan#icomoon') format('svg');
+  font-family: "icomoon";
+  src: url("fonts/icomoon.eot?nj6lan");
+  src:
+    url("fonts/icomoon.eot?nj6lan#iefix") format("embedded-opentype"),
+    url("fonts/icomoon.ttf?nj6lan") format("truetype"),
+    url("fonts/icomoon.woff?nj6lan") format("woff"),
+    url("fonts/icomoon.svg?nj6lan#icomoon") format("svg");
   font-weight: normal;
   font-style: normal;
   font-display: block;
 }
 
-[class^="icon-"], [class*=" icon-"] {
+[class^="icon-"],
+[class*=" icon-"] {
   /* use !important to prevent issues with browser extensions that change fonts */
-  font-family: 'icomoon' !important;
+  font-family: "icomoon" !important;
   speak: never;
   font-style: normal;
   font-weight: normal;
```

Comment:
Could you please exclude this specific file from the configuration? It's provided by Icomoon and we don't want to reformat it everytime we import a new version of the file. Thanks!

Reply:
✅